src/otx/algo/instance_segmentation/necks/fpn.py

```diff
@@ -162,3 +162,17 @@ def forward(self, inputs: tuple[Tensor]) -> tuple:
             for _ in range(self.num_outs - used_backbone_levels):
                 outs.append(torch.nn.functional.max_pool2d(outs[-1], 1, stride=2))
         return tuple(outs)
+
+
+class FPN:
```

Comment:
Could we remove this and instead use the FPN implementation in src/otx/algo/detection/necks/fpn.py to reduce duplication?

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/3865/commits/0f4409c90de69578b2e4b09e66f2b5b4c072e436